curvefs/src/mds/schedule/topoAdapter.h

```diff
@@ -116,9 +116,9 @@ struct MetaServerInfo {
     MetaServerInfo() : startUpTime(0) {}
     MetaServerInfo(const PeerInfo &info, OnlineState state,
                    const MetaServerSpace &space, uint32_t copysetNum = 0,
-                   uint32_t leaderNum = 0) : info(info), state(state),
-                   space(space), copysetNum(copysetNum), leaderNum(leaderNum),
-                   startUpTime(0) {}
+                   uint32_t leaderNum = 0) : startUpTime(0), info(info),
+                   state(state), copysetNum(copysetNum), leaderNum(leaderNum),
+                   space(space) {}
```

Comment:
There is no need to modify the initialization order.

Reply:
Eliminate compiled warnings

Reply:
warning: 'xxxxx' will be initialized after [-Wreorder]